client/jetpack-connect/test/authorize.js

```diff
@@ -66,7 +66,12 @@ const DEFAULT_PROPS = deepFreeze( {
 
 jest.mock( '@automattic/calypso-config', () => {
 	const mock = () => 'development';
-	mock.isEnabled = jest.fn( () => true );
+	mock.isEnabled = jest.fn( ( featureFlag ) => {
```

Comment:
We don't want to change the behavior of tests until we know for sure that we are going to deploy this to prod.